server/portal/settings/settings_custom.example.py

```diff
@@ -112,14 +112,6 @@
         'readOnly': False,
         'hideSearchBar': False
     },
-    {
-        'name': 'Google Drive',
-        'system': 'googledrive',
-        'scheme': 'private',
-        'api': 'googledrive',
-        'icon': None,
-        'integration': 'portal.apps.googledrive_integration'
-    }
```

Comment:
We can leave this in the example, as it's still the form we'd use in the future

Reply:
Readded to custom example